### ISSUE TITLE
xLights crashes on render.

### ISSUE THREAD

**Issue by magish01** [smeighan/xLights#1628]
**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
1. Render and save
2. Click Change starting location on "Garage" from bottom left to bottom right
3. Start render and crashes

**Expected behavior**
Render.

**Screenshots**

**Versions (please complete the following information):**  2019.37
[xLights_dbgrpt-8888-20190731T213055.zip](https://github.com/smeighan/xLights/files/3454907/xLights_dbgrpt-8888-20190731T213055.zip)

 - OS: [e.g. Windows 10, Mac 10.14.5, etc...]  Windows 10
 - xLights version [e.g. 2019.37]   2019.37

**Additional context**

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.

**Comment by AzGilrock** [comment]
What the hell is this.  It looks like instructions for reporting a bug.

**Comment by computergeek1507** [comment]
It is a template, that the user is supposed to fill out but clearly didn't.

**Comment by magish01** [comment]
Sorry I did not fill out your template correctly.  Just forget it and I will not report any crashes that I have in the future.  I obviously can't report problems in the EXACT manner that you want and don't appreciate the curt remarks I have received on the 2 crashes I have reported.  Sorry I wasted your time in trying to help report problems.  From here on out I will let others make the reports and I will just wait until corrections are made before I proceed with xLights.

**Comment by derwin12** [comment]
Your layout seems out of this world for channel and universe counts.. Are you sure that is correct?
~1 million channels?
~3600 Universes?

This is likely what is making it blow up..

**Comment by AzGilrock** [comment]
Yeah I've just never seen the template before.  Anyways I investigated this for a little while.  I was able to get the crash when I changed the start location for the garage model.  I tried to debug but there are other issues that make it really difficult.  For instance there is a model group called "PPD Snowflakes" which has effects on it but that group doesn't actually contain any models so it causes it to hit exception asserts in the code when debugging which makes it difficult to get to the real error.  Hopefully this might help someone dig deeper.
